### DISCUSSION TITLE
Failsafe defined in the receiver

### DISCUSSION THREAD

**Discussion by haplm** [ExpressLRS/ExpressLRS#2348] [Ideas]
With ELRS being more and more used in fixed wings which don't use a controller, would it be possible to add definition of failsafe values for the CRSF link in the receiver?

I know that it is possible to define failsafe values for PWM, but I'm feeding CRSF into some gyro, which translates it into PWM values. Yes, it should be probably the gyro which should allow me to set the failsafe values, but these gyros (or PWM converters generally) aren't the smartest thing on the planet.

Thank you!

**Comment by madrulzzz** [comment]
I agree, I have the exact same use case. It would be amazing if the Devs can add this feature.

**Comment by sk8board** [comment]
I asked for this a while ago and was told by a dev that it is planned for V3.5. https://github.com/ExpressLRS/ExpressLRS/discussions/2514

**Comment by sk8board** [comment]
You could use my project as a short term solution until ELRS 3.5 becomes available. 

https://github.com/sk8board/CRServoF_ELRS_for_RP2040_Pico

**Comment by sk8board** [comment]
Update: I was informed that Unified Failsafe was pushed back from ELRS v3.5 to 3.6. Therefore, we will have to wait for failsafe settings to be added to more than PWM ELRS receivers.

**Comment by haplm** [comment]
Thank you for the update. What would be lovely would be a  possibility to assign channel that could be used to simulate the failsafe condition to test the settings ☺️.
